pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -159,15 +161,17 @@ func (e *Signer) Run(ctx context.Context) {
 	defer utilruntime.HandleCrash()
 	defer e.syncQueue.ShutDown()
 
-	if !cache.WaitForNamedCacheSync("bootstrap_signer", ctx.Done(), e.configMapSynced, e.secretSynced) {
+	logger := klog.FromContext(ctx)
+	logger.Info("Starting", "controller", e.name)
+	defer logger.Info("Shutting down controller", "controller", e.name)
```

Comment:
This should be unified. Cc @pohly thoughts? 

```suggestion
	logger.Info("Starting", "controller", e.name)
	defer logger.Info("Shutting down", "controller", e.name)
```

or  

```suggestion
	logger.Info("Starting controller", "controller", e.name)
	defer logger.Info("Shutting down controller", "controller", e.name)
```

Reply:
we also log similar thing here:

https://github.com/kubernetes/kubernetes/blob/3694a02bb9b5c1dd216485d2e25bd2c8f24916dd/cmd/kube-controller-manager/app/controllermanager.go#L766

maybe it makes sense to differentiate it somehow?

Reply:
this is still unresolved, WDYT about the following proposal?

- let's do `Starting controller` and `Shutting down controller` in each controller
- change the `Starting controller` from the StartController function to `Initializing controller`